app/src/main/java/com/example/sabonit/Product.java

```diff
@@ -0,0 +1,84 @@
+package com.example.sabonit;
+
+/**
+ * Products of the Sabonit have to fit to a standard, ensuring they are fit to the format of the
+ * application. This is important for inserting them to the database properly and handling them.
+ * Every action with the product is done by the application developers. The users can get the
+ * information about the products in the ProductActivity.
+ *
+ * There are two kinds of attributes for every product: primary key attributes and product
+ * describing attributes. The primary key is a combination of attributes that are unique, for
+ * writing them in the database. The describing attributes are all the other attributes, and they
+ * are needed as important information of the product.
+ */
+public class Product
+{
+    /* ********* Attributes: ********* */
+    // The products are divided to categories, and every category is another department.
+    String department;
+    // The specific name of the product in the department.
+    String name;
+    // Price of one liter of the product, in use when computing the price of a given amount.
+    int price_per_liter;
+    // Optional, a short description or anything that not reflected in the product's department and name.
+    String description;
```

Comment:
You should consult with Efrat about it, because we need to combine this information in the design

Reply:
You are right! i'm on it :)